package.json

```diff
@@ -1,56 +1,58 @@
 {
-  "name" : "@scm-manager/scm-jenkins-plugin",
-  "private" : true,
-  "version" : "2.1.0-SNAPSHOT",
-  "license" : "MIT",
-  "main" : "./src/main/js/index.ts",
-  "scripts" : {
-    "build" : "ui-scripts plugin",
-    "watch" : "ui-scripts plugin-watch",
-    "postinstall" : "ui-plugins postinstall"
+  "name": "@scm-manager/scm-jenkins-plugin",
+  "private": true,
+  "version": "2.1.0-SNAPSHOT",
+  "license": "MIT",
+  "main": "./src/main/js/index.ts",
+  "scripts": {
+    "build": "ui-scripts plugin",
+    "watch": "ui-scripts plugin-watch",
+    "postinstall": "ui-plugins postinstall"
   },
-  "dependencies" : {
-    "@scm-manager/ui-plugins" : "2.0.0",
-    "@scm-manager/ui-components" : "^2.0.0",
-    "@scm-manager/ui-extensions" : "^2.0.0",
-    "classnames" : "^2.2.6",
-    "query-string" : "^5.0.1",
-    "react" : "^16.10.2",
-    "react-i18next" : "^10.13.1",
-    "react-redux" : "^5.0.7",
-    "react-router-dom" : "^5.1.2",
-    "redux" : "^4.0.0",
-    "styled-components" : "^4.4.0"
+  "dependencies": {
+    "@scm-manager/ui-plugins": "2.3.0-20200609-084213",
+    "@scm-manager/ui-components": "^2.3.0-20200609-084213",
+    "@scm-manager/ui-extensions": "^2.3.0-20200609-084213",
+    "classnames": "^2.2.6",
+    "query-string": "^5.0.1",
+    "react": "^16.10.2",
+    "react-i18next": "^10.13.1",
+    "react-redux": "^5.0.7",
+    "react-router-dom": "^5.1.2",
+    "redux": "^4.0.0",
+    "styled-components": "^5.1.0"
   },
-  "babel" : {
-    "presets" : [ "@scm-manager/babel-preset" ]
+  "babel": {
+    "presets": [
+      "@scm-manager/babel-preset"
+    ]
   },
-  "jest" : {
-    "preset" : "@scm-manager/jest-preset"
+  "jest": {
+    "preset": "@scm-manager/jest-preset"
   },
-  "prettier" : "@scm-manager/prettier-config",
-  "eslintConfig" : {
-    "extends" : "@scm-manager/eslint-config"
+  "prettier": "@scm-manager/prettier-config",
+  "eslintConfig": {
+    "extends": "@scm-manager/eslint-config"
   },
-  "devDependencies" : {
-    "@scm-manager/babel-preset" : "^2.0.0",
-    "@scm-manager/eslint-config" : "^2.0.0",
-    "@scm-manager/jest-preset" : "^2.0.0",
-    "@scm-manager/prettier-config" : "^2.0.0",
-    "@scm-manager/tsconfig" : "^2.0.0",
-    "@scm-manager/ui-scripts" : "^2.0.0",
-    "@scm-manager/ui-tests" : "^2.0.0",
-    "@scm-manager/ui-types" : "^2.0.0",
-    "@types/classnames" : "^2.2.9",
-    "@types/enzyme" : "^3.10.3",
-    "@types/fetch-mock" : "^7.3.1",
-    "@types/i18next" : "^13.0.0",
-    "@types/jest" : "^24.0.19",
-    "@types/query-string" : "5",
-    "@types/react" : "^16.9.9",
-    "@types/react-redux" : "5.0.7",
-    "@types/react-router-dom" : "^5.1.0",
-    "@types/styled-components" : "^4.1.19",
-    "jest" : "^24.9.0"
+  "devDependencies": {
```

Comment:
Do we need the suffix in the @scm-manager dependencies or would a simple "^2.3.0" suffice ?

Reply:
This is auto generated.